appeals/api/src/server/endpoints/appeals/appeals.controller.js

```diff
@@ -60,7 +60,10 @@ const getAppeals = async (req, res) => {
 	const formattedAppeals = await Promise.all(
 		appeals.map(async (appeal) => {
 			const linkedAppeals = await appealRepository.getLinkedAppeals(appeal.reference);
-			const commentCounts = await countAppealRepresentationsByStatus(appeal.id, 'comment');
+			const commentCounts = await representationRepository.countAppealRepresentationsByStatus(
+				appeal.id,
+				'comment'
+			);
```

Comment:
This seems like a weird thing to do. Can't we use primsa to populate the relations as the appeals are fetched form the db?

Reply:
Will create a ticket to look at this, possibly is a perf benefit.